decidim-core/spec/system/homepage_spec.rb

```diff
@@ -347,7 +347,7 @@
       describe "decidim link with external icon" do
         before { visit current_path }
 
-        let(:webpacker_helper) do
+        let(:shakapacker_helper) do
```

Comment:
We can remove this variable as it isn't used anymore. 
It was introduced at https://github.com/decidim/decidim/pull/8290, but then with the redesign this spec was changed as it isn't used.

Reply:
done in e429a57f36